client/trino-cli/src/main/java/io/trino/cli/ClientOptions.java

```diff
@@ -37,98 +44,165 @@
     public Optional<String> keystorePath;
 
+    @PropertyMapping(SSL_KEY_STORE_PASSWORD)
     @Option(names = "--keystore-password", paramLabel = "<password>", description = "Keystore password")
     public Optional<String> keystorePassword;
 
+    @PropertyMapping(SSL_KEY_STORE_TYPE)
     @Option(names = "--keystore-type", paramLabel = "<type>", description = "Keystore type")
     public Optional<String> keystoreType;
 
+    @PropertyMapping(SSL_TRUST_STORE_PATH)
     @Option(names = "--truststore-path", paramLabel = "<path>", description = "Truststore path")
     public Optional<String> truststorePath;
 
+    @PropertyMapping(SSL_TRUST_STORE_PASSWORD)
     @Option(names = "--truststore-password", paramLabel = "<password>", description = "Truststore password")
     public Optional<String> truststorePassword;
 
+    @PropertyMapping(SSL_TRUST_STORE_TYPE)
     @Option(names = "--truststore-type", paramLabel = "<type>", description = "Truststore type")
     public Optional<String> truststoreType;
 
+    @PropertyMapping(SSL_USE_SYSTEM_TRUST_STORE)
     @Option(names = "--use-system-truststore", description = "Use default system (OS) truststore")
     public boolean useSystemTruststore;
 
+    @PropertyMapping(SSL_VERIFICATION)
     @Option(names = "--insecure", description = "Skip validation of HTTP server certificates (should only be used for debugging)")
     public boolean insecure;
 
+    @PropertyMapping(ACCESS_TOKEN)
     @Option(names = "--access-token", paramLabel = "<token>", description = "Access token")
     public Optional<String> accessToken;
 
+    @PropertyMapping(USER)
     @Option(names = "--user", paramLabel = "<user>", defaultValue = "${sys:user.name}", description = "Username " + DEFAULT_VALUE)
     public Optional<String> user;
 
+    @PropertyMapping(PASSWORD)
     @Option(names = "--password", paramLabel = "<password>", description = "Prompt for password")
     public boolean password;
 
+    @PropertyMapping(EXTERNAL_AUTHENTICATION)
     @Option(names = "--external-authentication", paramLabel = "<externalAuthentication>", description = "Enable external authentication")
     public boolean externalAuthentication;
 
+    @PropertyMapping(EXTERNAL_AUTHENTICATION_REDIRECT_HANDLERS)
     @Option(names = "--external-authentication-redirect-handler", paramLabel = "<externalAuthenticationRedirectHandler>", description = "External authentication redirect handlers: ${COMPLETION-CANDIDATES} " + DEFAULT_VALUE, defaultValue = "ALL")
     public List<ExternalRedirectStrategy> externalAuthenticationRedirectHandler = new ArrayList<>();
 
-    @Option(names = "--source", paramLabel = "<source>", defaultValue = "trino-cli", description = "Name of source making query " + DEFAULT_VALUE)
-    public String source;
+    @PropertyMapping(SOURCE)
+    @Option(names = "--source", paramLabel = "<source>", description = "Name of source making query (default: " + SOURCE_DEFAULT + ")")
```

Comment:
maybe 

Name of the client to use as source that submits the query. Defaults to trino-cli.

Reply:
Done, but I didn't change the default in parentheses, since this needs to be consistent with all other options.